babi/file.py

```diff
@@ -673,39 +673,54 @@ def sort_selection(self, margin: Margin, reverse: bool = False) -> None:
     def _is_commented(self, lineno: int, prefix: str) -> bool:
         return self.buf[lineno].lstrip().startswith(prefix)
 
-    def _comment_remove(self, lineno: int, prefix: str) -> None:
-        line = self.buf[lineno]
+    def _start_space_offset(self, line: str) -> int:
         ws_match = WS_RE.match(line)
         assert ws_match is not None
-        ws_len = len(ws_match[0])
+        if '\t' in ws_match[0]:
+            return len([c for c in ws_match[0] if c != '\t'])
```

Comment:
if you wanted to do this a better way would be to not use `\s` in the regex (but don't do that because I think this approach is incorrect) the test output looks wrong, I would expect:

```
line_0
<tab># line_1
<tab># line_2
```

Reply:
Should be fixed in the last commit